src/components/Header/Header.css

```diff
@@ -0,0 +1,38 @@
+.header-container {
+  margin-top: 43px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+.header-container ul {
+  list-style-type: none;
+  display: flex;
+  -moz-column-gap: 35px;
+       column-gap: 35px;
+  align-items: center;
+}
+.header-container ul a {
+  font-weight: 500;
+  font-size: 15px;
+  color: #B8BECD;
+  font-family: "Inter", sans-serif;
+}
+.header-container ul .active a {
+  color: #202336;
+}
+.header-container ul .secondary-btn a, .header-container ul .primary-btn a {
+  width: 132px;
+  height: 44px;
+  text-align: center;
+}
```

Comment:
- [ ] Here as well, apply classnames for each of the elements instead of applying it directly to it.

Reply:
The changes were made in the scss but it dont compile the css . Fixed the issue